package.json

```diff
@@ -9,8 +9,7 @@
   "dependencies": {
     "postcss": "^4.1.11",
     "postcss-modules-extract-imports": "^0.0.5",
-    "postcss-modules-local-by-default": "^0.0.9",
-    "postcss-modules-scope": "^0.0.8"
```

Comment:
Should put the proper version of the scope plugin, when it will be updated.
https://github.com/css-modules/postcss-modules-scope/pull/15